### DISCUSSION TITLE
Usage of $navigationParentGroup

### DISCUSSION THREAD

**Discussion by JohnCarlo29** [filamentphp/filament#10496] [Help]
I have this sample filament resource class, I want to use the `$navigationParentGroup` by I'm getting an error `Call to a member function keyBy() on null` . Im trying to achieve to have this level of navigation `Shop->Setting->Users`

```class UserResource extends Resource
{
    protected static ?string $model = User::class;

    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';

    protected static ?string $navigationGroup = 'Setting';

    protected static ?string $navigationParentItem = 'Shop';
    }```

**Comment by leandrocfe** [comment]
Is the Shop a resource? Did you config the `$navigationGroup = 'Setting'` as well?
